tools/_setup_generation/step_designer.py

```diff
@@ -0,0 +1,73 @@
+# ##########################################################################################
+# Step to generate the documentation pages for Taipy Designer, after they were copied
+# locally from the taipy-designer repository.
+#
+# ##########################################################################################
+
+import os
+import re
+from io import StringIO
+
+from .setup import SetupStep, Setup
+
+
+class DesignerStep(SetupStep):
+    PREFIX = "userman/ecosystem/designer"
```

Comment:
What about changing the name to `DESIGNER_DOC_LOCATION`?

Reply:
Too long for me. This is very, very local.